amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectorSettings.scala

```diff
@@ -6,7 +6,10 @@ package akka.stream.alpakka.amqp
 
 import akka.annotation.InternalApi
 
+import java.util.{List => JavaList, Map => JavaMap}
```

Comment:
I think it is better to spell out the whole types where needed. In case a Java user jumps to the source, it would be easier to understand.